### ISSUE TITLE
cronjob "cron-proc-email" and few other resources doesn't take imageRegistry global parameter

### ISSUE THREAD

**Issue by SmartGuyy** [passbolt/charts-passbolt#56]
Hello,

I noticed few issues with this chart :
- some resources doesn't take "imageRegistry" from the global parameters
- cron-proc-email doesn't take neither "imageRegistry" from global nor the one from "app.image"

This is the values file i end up with so i can have "imageRegistry" correctly defined everywhere (except for cron-proc-email, where i couldn't find a solution --> i need to define manually after each deployment...) :

```
global:
  imagePullSecrets: 
    - xxx
  imageRegistry: ${CI_REGISTRY}
app:
  initImage:
    repository: ${MARIADB_IMAGE_REPO}
    tag: ${MARIADB_IMAGE_TAG}
    registry: ${CI_REGISTRY}
  image:
    repository: ${PASSBOLT_IMAGE_REPO}
    tag: ${PASSBOLT_IMAGE_TAG}
  cache:
    redis:
      enabled: true
      sentinelProxy:
        enabled: true
        image:
          repository: ${HAPROXY_IMAGE_REPO}
          tag: ${HAPROXY_IMAGE_TAG}
mariadb:
  image:
    repository: ${MARIADB_IMAGE_REPO}
    tag: ${MARIADB_IMAGE_TAG}
redis:
  auth:
  sentinel:
    enabled: true
    image:
      repository: ${REDIS_IMAGE_REPO}
      tag: ${REDIS_IMAGE_TAG}
      registry: ${CI_REGISTRY}
  image:
    repository: ${REDIS_IMAGE_REPO}
    tag: ${REDIS_IMAGE_TAG}
    registry: ${CI_REGISTRY}
```

**Comment by dlen** [comment]
Thanks for the heads up! We'll patch this one in the next release

**Comment by dlen** [comment]
Fixed in 0.5.0
